projectile.el

```diff
@@ -678,6 +679,13 @@ which we're looking."
              (setq file nil))))
     (and root (expand-file-name (file-name-as-directory root)))))
 
+(defvar-local projectile-custom-root nil
```

Comment:
I wasn't sure if that would be bad practice, to have a variable and a function with the same name.